docs/madr/decisions/070-resource-identifier.md

```diff
@@ -0,0 +1,99 @@
+# Resource Identifier
+
+* Status: accepted
+
+## Context and Problem Statement
+
+Kuma multizone mode allows synchronizing state between multiple clusters. Zone Control Planes connect to the Global Control Plane, forming a tree with the Global CP as the root.
+
+There are several resource synchronization patterns:
+- A resource is created on the Global CP and synced down to all Zone CPs.
+- A resource is created on a Zone CP and synced to the Global CP.
+- A resource is created on a Zone CP and synced to other Zone CPs.
+
+For example, the Zone CP in `zone-1` can possess the following MeshTimeouts:
+- MeshTimeouts created in `zone-1`
+- MeshTimeouts created on the Global CP and synced to `zone-1`
+- MeshTimeouts created in `zone-2` and synced to `zone-1`
+
+To uniquely identify a resource regardless of its place of origin, Kuma uses the following Go structure:
```

Comment:
In the ctx you need to specify where we're planning to use it. And then we need to check that the charset of this fits the places you are planning to use it in